elements/pfe-content-set/demo/index.html

```diff
@@ -1,486 +1,3 @@
-<!DOCTYPE html>
```

Comment:
Why was so much of this file removed?

Reply:
oof, I think this must have been a merge conflict. I'll bring it back.